### PR TITLE
Add Cmdlet and state it's not included

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/6655

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-powershell-release-notes.md [hunk 1]
```diff
@@ -85,6 +85,7 @@ This page provides the latest Teams PowerShell change log for both General Avail
 | [Set-TeamsApp](https://docs.microsoft.com/powershell/module/teams/set-teamapp?view=teams-ps) | Yes | Yes |
 | [Set-TeamTargetingHierarchy](https://docs.microsoft.com/powershell/module/teams/set-teamtargetinghierarchy?view=teams-ps) | Yes | **No** |
 | [Update-TeamsAppInstallation](https://docs.microsoft.com/powershell/module/teams/update-teamappinstallation?view=teams-ps) | Yes | **No** |
+| [Enable-CsOnlineSessionForReconnection](https://docs.microsoft.com/en-us/skypeforbusiness/set-up-your-computer-for-windows-powershell/diagnose-problems-with-the-skype-for-business-online-connector) | **No** | **No** |
```
Comment (JohanFreelancer9) [inline Teams/teams-powershell-release-notes.md:88]:
```suggestion
| [Enable-CsOnlineSessionForReconnection](https://docs.microsoft.com/skypeforbusiness/set-up-your-computer-for-windows-powershell/diagnose-problems-with-the-skype-for-business-online-connector) | **No** | **No** |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - please remove the locale from the link.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@brandber PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**